test_settings.py

```diff
@@ -1,10 +1,10 @@
 from gem.settings import *  # noqa: F401, F403
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'gem_test.db',
-    }
-}
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
+#         'NAME': 'gem_test.db',
+#     }
+# }
```

Comment:
I don't think this should be commented out should it? I'm guessing you did it for your local testing?

Reply:
@KaitCrawford Due to sqlite syntax issues when running migrations, I had to force test settings to use a postgres db

Reply:
@moh-moola did you add other db settings to get the migrations running locally? They aren't in this PR

Reply:
@KaitCrawford this is all that was needed

Reply:
@moh-moola I'm worried this is going to break things for the rest of us locally.
Because without these settings then the app will still use sqlite because of these settings https://github.com/praekeltfoundation/molo-gem/blob/develop/gem/settings/base.py#L255
Did you maybe set the `DATABASE_URL` setting in your local environment to use postgres?

Reply:
@KaitCrawford Nope, locally and on trav there are no test issues, I removed just the explicit declaration of the sqlite db conf